### DISCUSSION TITLE
How to add more modules? Mac OS ver

### DISCUSSION THREAD

**Discussion by ryogofreestyle** [DISTRHO/Cardinal#57] [General]
Hi, I would like to know if there is a way to add a module in MacOS.
Can I do it the same way as the vcv rack site?

**Comment by falkTX** [comment]
No you cannot (do it the same way as VCV modules).
There is already a discussion about this, see #28 please continue discussion there

The latest commit in this repo shows me adding repelzen https://github.com/DISTRHO/Cardinal/commit/b8f2e64e32da31bd67f42a65cfd88573e475bcf9
